utam-compiler/src/main/java/utam/compiler/translator/TranslatorGenerationCommand.java

```diff
@@ -43,13 +43,18 @@
   static final String OUTPUT_DIRECTORY_MISSING = "Output directory is not configured";
   static final String PACKAGE_CONFIG_MISSING = "Packages mapping is not configured";
   static final String REDUNDANT_CLI_ARGS = "If JSON file is set, all other arguments are ignored";
+  static final String ERR_COMPILER_CONFIG_NEEDS_ROOT = "To resolve configuration paths, set compilerRoot";
 
-  @Option(names = {"-c", "-config", "--config"},
+  @Option(names = {"-g", "-config", "--config"},
```

Comment:
Maybe we make this option `--configFile` with a shortcut of `-f`, and change the `--compilerRoot` shortcut to `-c`? It's early enough in this feature we can change it without hurting anything. Later on, we'll probably want to remove the other configuration command-line options.

Reply:
done and done. thx.